### PR TITLE
Privacy

### PR DESCRIPTION
**Pull request by jonnyarnold**
As requested by @JohnPlummer, an article outlining what information we track, how we use it and why we do it.

To be reviewed by @JohnPlummer.

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
*If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md).*

Good code is **tested**, **designed for change**, **easy to understand** and **able to cope when
things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

File: docs/tracking-privacy.markdown [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+We also use [Google Analytics](https://www.google.co.uk/analytics/) to send and store some messages. Similar to ReevooMark, Google Analytics puts a JavaScript library on the website and provides the ability to send messages back to Google's database for use by Reevoo and the companies that use Reevoo.
+
+When you first visit a website, Reevoo and Google Analytics will set [cookies](http://www.bbc.co.uk/privacy/cookies/about) on your device. Their names start with *reevoo* and allow us to identify you between web pages by a unique number. *(For more information on why we want to identify you between web pages, see the following section.)*
+
+
+## What We Track and Why We Track It
+
+**We want to give users the information they want easily.** In order to do this, we need to understand how users behave when using Reevoo's services. To do this, we track information on what users do and don't use. This involves tracking clicks on Reevoo content.
```
Comment (shyvy88) [inline docs/tracking-privacy.markdown]: To do this used twice in two consecutive sentences. To do this better I would change one of the sentences not to include it.

File: docs/tracking-privacy.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Every user has a different way of using our customers' websites. **We want to make sure that users can use Reevoo's services**, irrespective of their browser or device. To do this, we test out new features on a representative sample of devices and browsers. To determine what a "representative sample" is, we track information on what browsers and devices our users are using. We also track error messages, so we can investigate the issue and fix it for the future.
+
+**We need to prove to our customers that we are valuable to them.** They usually have asked for Reevoo's services in order to improve sales or spend longer on their website. In order to prove that we do this, we track information on how long users spend reading reviews. We also track when users read reviews and then go on to purchase the product.
```
Comment (shyvy88) [inline docs/tracking-privacy.markdown]: to improve sales or *the time users spend* on their website

File: docs/tracking-privacy.markdown (hunk 1 above)
Comment (jonnyarnold) [inline docs/tracking-privacy.markdown]: Sorted :+1:

File: docs/tracking-privacy.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+**Reevoo does not sell tracking information to anyone,** and we do not give your information to advertisers. 
+
+We *do not* use tracking information to filter what you see from our services.
+
+
+## Controlling Tracking
+
+If you do not wish for your information to be used, you can take the following measures:
+
+- If you have an ad blocker in your browser, add `t.mark.reevoo.com` to your ad-blocking blacklist. This is our dedicated tracking URL, and all other features of ReevooMark will still work.
```
Comment (JohnPlummer) [inline docs/tracking-privacy.markdown]: I think we should add the snowplow URL also

**Comment by jonnyarnold** [comment]
@JohnPlummer Ready to go?

**Comment by reevoo-samuel** [comment]
I don't see any comments on your Pull Request.

Are you too good for code reviews now?
No. No you aren't.

Maybe you forgot to say who reviewed it. That's fine, but let me know.

**Comment by jonnyarnold** [comment]
There were loads, Samuel. I'll fix you on Monday.

File: docs/tracking-privacy.markdown [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+**We *do not* track any information that is linked to an individual**, such as a name or address. When you first visit a site you are given a unique number, and everything we track uses this number only.
+
+
+## What We Do With The Information
+
+The information that we collect is stored in our database or in Google Analytics, depending on the service used to send the message. This data is usually aggregated as it comes into the database to reduce the amount of storage space it takes up.
+
+This information is used within Reevoo for the reasons described in the sections above. Our customers also have access to this information for their own analysis.
+
+**Reevoo does not sell tracking information to anyone,** and we do not give your information to advertisers.
```
Comment (edwininlondon) [inline docs/tracking-privacy.markdown]: Hmm sounds a bit too strict. We might charge our clients extra for telling them a bit more about the people who are on their site and use Reevoo. We're not selling that data to random adevrtisers that is true.

File: docs/tracking-privacy.markdown [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+## What We Do With The Information
+
+The information that we collect is stored in our database or in Google Analytics, depending on the service used to send the message. This data is usually aggregated as it comes into the database to reduce the amount of storage space it takes up.
+
+This information is used within Reevoo for the reasons described in the sections above. Our customers also have access to this information for their own analysis.
+
+**Reevoo does not sell tracking information to anyone,** and we do not give your information to advertisers. 
+
+We *do not* use tracking information to filter what you see from our services.
+
```
Comment (edwininlondon) [inline docs/tracking-privacy.markdown]: Hmm ...  I assume that we want to convey that we're showing all reviews and you can trust our content. Yet I could see a case where we show a people like you set of reviews, with a link to see all, in order to help the user deal with the noise.

**Pull request merged**